extensions/sonarcloud/overview.md

```diff
@@ -1,18 +1,18 @@
 **[SonarCloud][sc]** is the leading product for Continuous Code Quality online, totally free for open-source projects. It supports all major programming languages, including C#, VB .Net, JavaScript, TypeScript, C/C++ and many more. If your code is closed source, SonarCloud also offers a paid plan to run private analyses.
 
-This VSTS extension provides build tasks that you can add in your build definition. All branches and pull-requests are automatically analyzed, allowing you to discover early any bug or vulnerability in the code. A widget is also available to track the health of the overall application.
+This Azure DevOps extension provides build tasks that you can add in your build definition. All branches and pull-requests are automatically analyzed, allowing you to discover early any bug or vulnerability in the code. A widget is also available to track the health of the overall application.
 
 To get started in a few minutes, you can follow this dedicated [Microsoft Lab][getstarted].
 
-## About the SonarCloud VSTS Marketplace Extension
+## About the SonarCloud Azure DevOps Marketplace Extension
```

Comment:
Just a note: I feel it's OK to replace VSTS by just "Azure DevOps" even if it should be "Azure DevOps Services" if we wanted to be really accurate. (because unlike the SQ extension, the SC one can be installed only on VSTS)

Reply:
Agreed, even more so that the [Works with](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud) takes care of mentioning `Azure DevOps Services` for us. It's nice like that, lets us focus on SonarQubeVSSonarCloud, and let the extension properties give further details if needed.

Reply:
Agreed, even more so that the [Works with](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud) takes care of mentioning `Azure DevOps Services` for us. It's nice like that, lets us focus on SonarQubeVSSonarCloud, and let the extension properties give further details if needed.